libs/platform/src/lib/components/form/checkbox/checkbox.component.html

```diff
@@ -0,0 +1,14 @@
+<fd-checkbox
```

Comment:
Also I see only checkbox here but how this is going to work with FormField and FormGroup?  It needs to be part of it, so in formField you set label for the field ..

Reply:
have included <div fd-form-item> in template.